hummingbot/connector/exchange/hitbtc/hitbtc_websocket.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+import asyncio
+import copy
+import logging
+import websockets
+import json
+from hummingbot.connector.exchange.hitbtc.hitbtc_constants import Constants
+
+
+from typing import (
+    Any,
+    AsyncIterable,
+    Dict,
+    Optional,
+)
+from websockets.exceptions import ConnectionClosed
+from hummingbot.logger import HummingbotLogger
+from hummingbot.connector.exchange.hitbtc.hitbtc_auth import HitbtcAuth
+from hummingbot.connector.exchange.hitbtc.hitbtc_utils import RequestId
+
+# reusable websocket class
+# ToDo: We should eventually remove this class, and instantiate web socket connection normally (see Binance for example)
+
+
+class HitbtcWebsocket(RequestId):
```

Comment:
Is there a particular reason for creating a `WebSocket` class for HitBTC?

Reply:
@CrimsonJacket not really no, I just thought that was the desired approach as per `crypto_com` and it made the code a bit cleaner since it's shared

Reply:
I see. Well it isn't really much of an issue, its just that we didn't intend to include the `Websocket` class for `crypto_com`.